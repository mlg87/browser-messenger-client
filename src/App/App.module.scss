.container {
    align-items: center;
    display: flex;
    flex: 1;
    font-size: 16px;
    height: 100vh;
    justify-content: center;
}