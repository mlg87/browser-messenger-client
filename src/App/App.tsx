import { Provider } from 'mobx-react';
import React from 'react';
import { Root } from '../routes';
import appState from '../stores';

const App: React.FunctionComponent = () => (
    <div>
        <Provider {...appState}>
            <Root />
        </Provider>
    </div>
);

export default App;
