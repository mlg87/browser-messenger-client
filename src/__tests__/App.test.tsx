import { render } from '@testing-library/react';
import React from 'react';
import App from '../App/App';

const setup = () => render(<App />);

describe('App', () => {

    it('renders', () => {

        const { container, debug } = setup();
        expect(container.firstChild).toMatchSnapshot();

    });

});
