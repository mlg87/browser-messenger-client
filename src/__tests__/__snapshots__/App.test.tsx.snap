// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <h1>
        welcome. let's do some chatting.
      </h1>
      <span
        class="link-container"
      >
        <a
          href="/login"
        >
          login
        </a>
        <a
          href="/register"
        >
          register
        </a>
      </span>
    </div>
  </div>
</div>
`;
